code/datums/skills/_skill.dm

```diff
@@ -1,10 +1,34 @@
 /datum/skill
-	var/name = "Skill"
+	var/name = "Skilling"
+	var/title = "Skiller"
 	var/desc = "the art of doing things"
 	var/modifiers = list(SKILL_SPEED_MODIFIER = list(1, 1, 1, 1, 1, 1, 1)) //Dictionary of modifier type - list of modifiers (indexed by level). 7 entries in each list for all 7 skill levels.
+	var/skillcapepath
+	var/list/levelUpMessages = list()
+	var/list/levelDownMessages = list()
 
 /datum/skill/proc/get_skill_modifier(modifier, level)
 	return modifiers[modifier][level+1] //+1 because lists start at 1
+/**
+  * new: sets up some lists. 
+  *
+  *Can't happen in the datum's definition because these lists are not constant expressions
+  */
+/datum/skill/New()
+	. = ..()
+	levelUpMessages = list("<span class='nicegreen'>I'm starting to figure out what [name] really is!</span>",
+	"<span class='nicegreen'>I'm getting a little better at [name]!</span>",
+	"<span class='nicegreen'>I'm getting much better at [name]!</span>",
+	"<span class='nicegreen'>I feel like I've become quite proficient at [name]!</span>",
+	"<span class='nicegreen'>After lots of practice, I've begun to truly understand the intricies \
+	 and surprising depth behind [name]. I now condsider myself a master [title].</span>",
+	"<span class='nicegreen'>Through incredible determination and effort, I've reached the peak of my [name] abiltities. I'm finally able to consider myself a legendary [title]!</span>" )
+	levelDownMessages = list("<span class='nicegreen'>I'm starting to forget what [name] really even is. I need more practice...</span>",
+	"<span class='nicegreen'>I'm getting a little worse at [name]. I'll need to keep practicing to get better at it...</span>",
+	"<span class='nicegreen'>I'm getting a little worse at [name]...</span>",
+	"<span class='nicegreen'>I'm losing my [name] expertise ....</span>",
+	"<span class='nicegreen'>I feel like I'm losing my mastery of [name].</span>",
+	"<span class='nicegreen'>I feel as though my legendary [name] skills have deteriorated. I'll need more intense training to recover my lost skills.</span>" )
```

Comment:
Put these in the list var definitions.

Reply:
Do you mean putting these list definitions on line 7 and 8? 
![image](https://user-images.githubusercontent.com/29008542/80250903-adcbce80-8642-11ea-8924-93aa0ed2566a.png)

Reply:
Because the lists are non-constant expressions and shouldn't go there

Reply:
```cpp
var/levelUpMessages = list("<span class='nicegreen'>I'm starting to figure out what [name] really is!</span>",
	"<span class='nicegreen'>I'm getting a little better at [name]!</span>",
	"<span class='nicegreen'>I'm getting much better at [name]!</span>",
	"<span class='nicegreen'>I feel like I've become quite proficient at [name]!</span>",
	"<span class='nicegreen'>After lots of practice, I've begun to truly understand the intricies \
	 and surprising depth behind [name]. I now condsider myself a master [title].</span>",
	"<span class='nicegreen'>Through incredible determination and effort, I've reached the peak of my [name] abiltities. I'm finally able to consider myself a legendary [title]!</span>" )
```
What benefit are you aiming for by doing it in `Initialize()`?

Reply:
uuh its literally impossible to put them in the list definitions because they are non-constant expressions

Reply:
unless i'm missing something

Reply:
You can do it, try it.

Reply:
![image](https://user-images.githubusercontent.com/29008542/80254850-e8853500-8649-11ea-9fd9-3d75fc32c102.png)

Reply:
Name it `var/list/levelUpMessage`

Reply:
What? You know that doesn't change it from being a non-constant expression, right?

Reply:
(typing it as a list doesn't change anything, btw)

Reply:
(just to be sure, i gave it an explicit list type and it didn't change anything)

Reply:
rohesie smoked too much of the weed, put it in init

Reply:
Right, it would have helped if I actually looked at the contents of the list.
Ignore everything I said.